cmd/git-sync/main.go

```diff
@@ -129,7 +135,8 @@ var flHTTPMetrics = flag.Bool("http-metrics", envBool("GIT_SYNC_HTTP_METRICS", t
 var flHTTPprof = flag.Bool("http-pprof", envBool("GIT_SYNC_HTTP_PPROF", false),
 	"enable the pprof debug endpoints on git-sync's HTTP endpoint")
 
-var log *customLogger
+var cmdRunner *cmd.CommandRunner
+var logger *logging.Logger
```

Comment:
rather than change every log-site, can we just leave it as "log"

Reply:
reverted back to log.